user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_VHDX.md

```diff
@@ -0,0 +1,155 @@
+
+1. [Create the VM](#creating-the-vm).
+1. [Connect and start the VM](#connecting-and-starting-the-vm).
+1. [Configure DataMiner](#configuring-dataminer).
+1. Optionally, [switch from subscription mode to a perpetual-license setup](#switching-from-subscription-mode-to-perpetual-license).
+
+## Creating the VM
+
+When you have downloaded the VHDX, you can start to create a VM in your chosen virtualization environment. Below you can find the steps to follow in Hyper-V:
+
+1. Start creating your VM by following [the official Hyper-V guide](https://learn.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/create-a-virtual-machine-in-hyper-v).
+
+1. Specify a name for your VM and store the machine in a location of your own choice, making sure your disk has enough space.
+
+1. Choose Virtual Machine Generation 2.
+
+1. Assign (at least) 8192 MB startup memory.
+
+   Optionally, you can choose to use dynamic memory.
+
+   > [!IMPORTANT]
+   > If you intend to run the databases locally on the same server using WSL, assign at least 12288 MB RAM, as the WSL storage container will take about 4 GB to run.
+
+1. Connect to a virtual switch that has internet access (either the default switch or a custom one).
+
+   > [!NOTE]
+   > If you choose the default virtual switch, you will only be able to use [remote access](xref:Cloud_Remote_Access) through dataminer.services to access the system outside of the virtual machine environment.
+
+1. Connect the virtual hard disk that you have just downloaded.
+
+> [!IMPORTANT]
+> If you intend to use the locally hosted Cassandra and OpenSearch clusters running on WSL, make sure to enable [nested virtualization](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization) on your host PC for the VM you created before starting the VM. To do so, run the following command on your host PC in an elevated Powershell prompt:
+>
+> ```powershell
+> Set-VMProcessor -VMName <VMName> -ExposeVirtualizationExtensions $true
+> ```
+
+## Connecting and starting the VM
+
+1. When you have created the VM, double-click the entry and then click *Start* to boot the VM.
+
+   You will see that the VM will boot in the OOBE setup screen.
+
+1. Choose the region and keyboard settings and select a strong password for the built-in Administrator account.
+
+   When you have set the password, the VM will restart.
+
+1. When the login screen is shown, log in to the VM.
+
+## Configuring DataMiner
+
+As soon as you log in to the VM, a window will be shown where you can configure your DataMiner System.
+
+![Configuring DataMiner Agent Screen](~/user-guide/images/install-first-startup-choice-script.png)
+
+> [!IMPORTANT]
+>
+> - If you intend to restore a backup coming from another machine because of e.g. a hardware migration or during disaster recovery, skip the configuration below and follow the steps to [obtain a DataMiner license](xref:DataminerLicenses).
```

Comment:
This is a little confusing, as this hyperlink refers to a procedure where users still need install DataMiner. I think we'll need to add more detail here in addition to (or instead of) the hyperlink.

Reply:
Perhaps it would then be better to add another procedure just like 'Switching from subscription mode to perpetual license'. Because it is indeed not a default procedure, but related to this pre-installed DataMiner setup. Let me know if you agree and I will add it.

Reply:
Sounds like a good idea, go ahead!

Reply:
Added procedure